### PR TITLE
Pr/719

### PR DESCRIPTION
**Pull request by dzucconi**
Re: https://github.com/aredotna/ervell/pull/719

### CODE REVIEW

File: models/user.coffee [hunk 1]
```diff
@@ -22,6 +22,14 @@ module.exports = class User extends Model
   isPremium: ->
     @get('is_premium')
 
+  homePath: ->
+    if parseInt(@get('following_count')) <= 1 then '/explore' else (@get('home_path') or '/')
+
+  hasCustomHomePath: ->
+    @get('home_path') != '/explore' and
+    @get('home_path') != '/' and
+    @get('home_path') != "/#{@get('slug')}"
```
Comment (dzucconi) [inline models/user.coffee]: oh lol

Comment (broskoski) [inline models/user.coffee]: hm?

Comment (dzucconi) [inline models/user.coffee]: https://github.com/aredotna/ervell/pull/723/commits/1ea2f2d1c7258195848e75a68d425610b92a22cc#diff-0249de556acb4d3659af85ff67643708R2

also it's not used anyway. gonna delete it

File: apps/settings/components/your_plan/index.jade [hunk 2]
```diff
@@ -3,47 +3,50 @@
...
+
+            option(
+              value='premium_1'
+              selected= (customer.get('plan_id') === 'premium_1') ? 'selected' : null
+            ) Premium $45 / year
 
-      unless customer.get('is_lifetime')
-        hr
+    if customer.has('plan_id') && !customer.get('is_lifetime')
+      //- hr.Base
```
Comment (broskoski) [inline apps/settings/components/your_plan/index.jade]: Take this out?

Comment (dzucconi) [inline apps/settings/components/your_plan/index.jade]: woops

**Comment by broskoski** [comment]
LGTM, but lets do at least a little bit of validation...maybe check for a `/` at the beginning at least?

**Comment by dzucconi** [comment]
Nah we don't need to. It works.

**Comment by dzucconi** [comment]
Cleaning up a few last things.

**Pull request merged**